core-services/prow/02_config/openshift/machine-api-provider-gcp/_prowconfig.yaml

```diff
@@ -54,6 +54,8 @@ tide:
     - main
     - master
     - release-4.10
+    - release-4.11
+    - openshift-4.11
```

Comment:
@petr-muller is that what you meant by adding to complement query?

Reply:
Yes. Check out https://docs.google.com/document/d/19kxmzXFnXrbLChZXBfxy68mCYg06j3qT93VF23wepog/edit#heading=h.nzgnzwm8ihp5 - the [complement](https://en.wikipedia.org/wiki/Complement_(set_theory)) query should basically contain "everything else", so its `excludedBranches` list needs to contain all branhces that are included in all other queries.